src/server/static/js/lines.js

```diff
@@ -84,10 +63,12 @@ function getListOfScenarios(activeData) {
   return Object.keys(activeData)
 }
 
+var url_string = window.location.href;
+var url = new URL(url_string);
+var channel = url.searchParams.get('channel')
 
 
-
-d3.json("https://storage.googleapis.com/static-covid/static/data-staging-gleam.json")
+d3.json("https://storage.googleapis.com/static-covid/static/data-" + (channel ? channel : 'main') +"-gleam.json")
```

Comment:
I think this would be simpler with a template string:

```js
d3.json(`https://storage.googleapis.com/static-covid/static/data-${channel ? channel : 'main'}-gleam.json`)
```

Is there a reason we aren't using them? Do we need to make this codebase ES5-compatible?